Modules/posixmodule.c

```diff
@@ -9115,16 +9115,104 @@ os_setpgrp_impl(PyObject *module)
 #ifdef HAVE_GETPPID
 
 #ifdef MS_WINDOWS
-#include <processsnapshot.h>
```

Comment:
I think this header file is still required because we are using `PssCaptureSnapshot` bellow.

Reply:
Indeed it is - missed that. Is there a way I can add commits to the PR without making a new one? On other repos just committing to my own branch worked.

Reply:
You can just add new commit to your branch, and this PR will be updated. Close a PR and create new one will make the review work harder.